README.md

```diff
@@ -125,11 +125,16 @@ steps:
       cache-dependency-path: subdir/go.sum
   - run: go run hello.go
   ```
-## Getting go version from the go.mod file
-
-The `go-version-file` input accepts a path to a `go.mod` file or a `go.work` file that contains the version of Go to be used by a project. As the `go.mod` file contains only major and minor (e.g. 1.18) tags, the action will search for the latest available patch version sequentially in the runner's directory with the cached tools, in the [version-manifest.json](https://github.com/actions/go-versions/blob/main/versions-manifest.json) file or at the go servers.
+## Getting go version from a file
 
 If both the `go-version` and the `go-version-file` inputs are provided then the `go-version` input is used.
+
+If the file contains only major and minor (e.g. 1.18) tags, the action will search for the latest available patch version sequentially in the runner's directory with the cached tools, in the [version-manifest.json](https://github.com/actions/go-versions/blob/main/versions-manifest.json) file or at the go servers.
+
+### If the go.mod file
+
+The `go-version-file` input accepts a path to a `go.mod` file or a `go.work` file that contains the version of Go to be used by a project. As the `go.mod` file contains only major and minor (e.g. 1.18) tags, the action will follow the above-mentioned approach.
```

Comment:
Could be useful to mention the `go-version` output here https://github.com/actions/setup-go/blob/41c2024c46acfe1d0b8c9b7f20e28406983e553b/action.yml#L23

as it makes it easier to know what go version was resolved to from the `go.mod` file

Reply:
See https://github.com/actions/setup-go/pull/295/commits/fde5dd24bd8d1dfc2a055288385cb9500846aa93